opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1561,7 +1561,7 @@ static int create_ep(opal_btl_usnic_module_t* module,
                        rc, fi_strerror(-rc));
         return OPAL_ERR_OUT_OF_RESOURCE;
     }
-    rc = fi_ep_bind(channel->ep, &module->av->fid, FI_RECV);
+    rc = fi_ep_bind(channel->ep, &module->av->fid, NULL);
```

Comment:
This is going to disappear when we PR/merge this to master -- when you rebase this branch, it should disappear from this commit automatically.